static/app/views/performance/landing/chart/utils.tsx

```diff
@@ -0,0 +1,3 @@
+export function getFieldOrBackup(field: string, backupField?: string) {
```

Comment:
this is the only util function still being used. moved it next to the places using it